src/Dapr.AspNetCore/Subscription.cs

```diff
@@ -0,0 +1,44 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Dapr
+{
+    /// <summary>
+    /// This class defines subscribe endpoint response
+    /// </summary>
+    public class Subscription
```

Comment:
These types need to be internal, they are not interesting for users of the SDK. Or if you like, convert back to manual JSON which I kinda prefer.

Reply:
IMO, strongly typed subscription can be easily managed in future when the metadata object grows with more properties. So marking it as internal as you suggested.